package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "willow-spinkit",
-  "version": "8.0.0",
+  "version": "8.0.1",
```

```suggestion
  "version": "9.0.0",
```
Because the Willow version is breaking backwards compatibility